projects/sensory_interface/index.html

```diff
@@ -9,9 +9,18 @@
 <body>
     <textarea id="textInput">1	2	3
 4	5	6</textarea>
-    <button id=" processButton" onclick="processData()">process </button>
```

Comment:
Good. ID is only needed if you want to obtain the element by ID, or reference it by ID from the CSS.